### ISSUE TITLE
Game reverts resolution in higher than native. DSR/DLDSR

### ISSUE THREAD

**Issue by r3538987** [ThirteenAG/GTAIV.EFLC.FusionFix#597]
### Game version this happens in

v1.2.0.59

### What area of the game

My native resolution is 1440p.
DLDSR is 2160p (4K), and DSR is 2880p (5K).

Non-CE versions do accept these with ease, and keep resolutions even after restart. 
In CE when higher resolutions are applied, screen flickers multiple times, and resolution goes back to 1440p. 

Few times managed to apply high resolution, after toggling Windowed to ON and OFF. 
But once such resolution is set, after restart, game again launches in 1440p.

Also noticed that in non-CE screenshots created in higher resolutions have proper file dimensions.
In CE, despite resolution, game always outputs screenshots in native res. 

DXVK is applied. 

Share your experience, maybe some one of had encountered similar behavior. 

### What should happen

Game should maintain previously selected resolution even after game restart.

### Checklist

- [X] Test in the [latest git build](https://ci.appveyor.com/api/projects/ThirteenAG/gtaiv-eflc-fusionfix/artifacts/GTAIV.EFLC.FusionFix.zip) in case it's already fixed.
- [X] [Search for other reports](https://github.com/ThirteenAG/GTAIV.EFLC.FusionFix/search?q=my+issue&type=issues) of the same issue.
- [X] Include logs or screenshots of issue.

**Comment by Ash-735** [comment]
This must be a DXVK issue as I'm using DLDSR to get higher resolutions on mine and the game remembers it in DX9. If I remember correctly one of the things needed to be done for DXVK was setting the resolution in the ini for it to be detected so perhaps that is set to your monitors native resolution and the game keeps defaulting back to that, perhaps try putting the DLDSR resolution in that space instead and see how the game reacts.

**Comment by r3538987** [comment]
1) This does not happen with non-CE, with DXVK or without. 
2) Removed DXVK for CE, and behavior remains. Such thing I did noticed for long time, maybe this is due to mine monitor setup.

**Comment by r3538987** [comment]
If I set main display from 1440p to 2160p DLDSR, game does start to accept 2160p resolution, plus switching to 5K also works. 
Quite odd. 

But after restart it reverts to 2160p from 5K still.

**Comment by Ash-735** [comment]
> If I set main display from 1440p to 2160p DLDSR, game does start to accept 2160p resolution, plus switching to 5K also works. Quite odd.
> 
> But after restart it reverts to 2160p from 5K still.

Yeah it's a DXVK issue, the game will always boot into what the ini says, so it's just best to set the resolution in the DXVK ini and leave it at that.

**Comment by RecklessGlue540** [comment]
I believe this issue should be opened to DXVK's repo as it's probably something either not implemented there or not working, or buggy.
But for example I wouldn't have this issue with AMD VSR on my part, as it reports as high as 8k correctly and those are selectable in game as well flawlessly. I believe even on DX9 I have no issues, so may be something with Nvidia's doings? not sure.

> Yeah it's a DXVK issue, the game will always boot into what the ini says, so it's just best to set the resolution in the DXVK ini and leave it at that.

Also if you allow me to make a slight correction, setting resolution through the dxvk.conf is not possible in DXVK, however you can force a specific aspect ratio that will in turn only report 16:9 resolutions for example, but I'm not so sure either as I believe that has been made obsolete, iirc as of 2.4, i don't remember the commit
in any case the command is this:
`d3d9.forceAspectRatio = i.e 4:3, 16:9 etc`
feel free to test, although it may not be helpful and in that case should be reported to their repository... I would report it either way though..

**Comment by r3538987** [comment]
> I believe this issue should be opened to DXVK's repo as it's probably something either not implemented there or not working, or buggy. 

Kind'a pointless, if game behaves same even without it.

**Comment by r3538987** [comment]
**And now after not touching game for days it launches and keeps resolution, like Ash said. 
5k/4k, and it remains after restart, with DXVK.** 

Maybe my setup is slightly strange, with 2 or 3 displays. And if 3rd (TV) is powered on PC sends to it 4K image. 
Could be 1080p@60 + 2160p@60 + pushing 4K/5K on another monitor adds up. 
Maximum digital resolution usually is 8K, and it's per port. Likely some strange factors plays here.
